analyses/tp53_nf1_score/03-plot-distribution-scores.Rmd

```diff
@@ -0,0 +1,121 @@
+---
+title: "Plot scores for TP53 inactivation"
+output: html_notebook
+---
+
+### Description
+This script uses the tp53 inactivation from the classifer
+and plots the distribution of tp53 status for both stranded and polya.
+Additional barplots also visualize distribution for samples with high
+tp53 inactivation without tp53 mutations/losses.
+
+
+Read in scores from tp53-nf1-classifier and tp53 status file used for the evaluation
+```{r}
+
+#rootdir
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+analyses_dir<-file.path(root_dir,"analyses","tp53_nf1_score")
+data_dir<-file.path(root_dir,"data")
+
+library("ggpubr")
+library("ggthemes")
+library("tidyverse")
+score_stranded_df<-read_tsv(file.path(analyses_dir,"results","score_status_stranded.tsv")) %>% dplyr::select("TP53_SCORE","tp53_status","sample_id","Kids_First_Biospecimen_ID","Kids_First_Participant_ID") %>% unique()
+
+score_stranded_long<-reshape2::melt(score_stranded_df,id=c("tp53_status","sample_id","Kids_First_Biospecimen_ID","Kids_First_Participant_ID")) 
+score_stranded_long$tp53_status<-factor(score_stranded_long$tp53_status)
+
+score_polya_df<-read_tsv(file.path(analyses_dir,"results","score_status_polya.tsv")) %>% dplyr::select("TP53_SCORE","tp53_status","sample_id","Kids_First_Biospecimen_ID","Kids_First_Participant_ID") %>% unique()
+
+score_polya_long<-reshape2::melt(score_polya_df,id=c("tp53_status","sample_id","Kids_First_Biospecimen_ID","Kids_First_Participant_ID")) 
+score_polya_long$tp53_status<-factor(score_polya_long$tp53_status)
+
+
+histology<-read_tsv(file.path(data_dir,"pbta-histologies.tsv"))
```

Comment:
This file has had some parsing issues due to recent-ish data release, so we're adding the argument `guess_max = 10000` to `read_tsv` as described the draft PR [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/697). Quick change before we get too far into this PR...